internal/api/api.go

```diff
@@ -216,6 +219,27 @@ func (api *idler) Info(w http.ResponseWriter, r *http.Request, ps httprouter.Par
 	}
 }
 
+func (api *idler) Reset(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+	openShiftAPI, openShiftBearerToken, err := api.getURLAndToken(r)
+	if err != nil {
+		log.Error(err)
+		w.WriteHeader(http.StatusBadRequest)
```

Comment:
why is this a bad request if fetching token fails?

Reply:
getURLAndToken fails only when `OpenShiftAPIParam` is not passed or it value is incorrect(that's when GetToken() fails), making it a bad request. We have only limited number of OpenShift API URLs. I think only 2, one for each cluster.

Reply:
What happens in case of n/w error or other transient errors?